com.github.rssguard.json

```diff
@@ -21,7 +21,7 @@
                 {
                     "type": "git",
                     "url": "git://github.com/martinrotter/rssguard.git",
```

Comment:
```suggestion
                    "url": "https://github.com/martinrotter/rssguard.git",
```

Reply:
i see, i had literally no idea how to fix that